File: patterns/pattern-7/pattern7.py
n = int(input())
for i in range(1,n+1):
    # for loop for spaces
    for j in range(1,(n-i + 1)+1):
        print(" ",end=" ")
    # for loop for star
    for k in range(1,2*i):
        print('*',end=" ")
    print()

File: patterns/pattern-7/pattern7.js
process.stdin.on('data',(userInput)=>{
    n = parseInt(userInput)
    for(let i = 1;i<=n;i++){
        // for loop for spaces
        for(let j = 1;j<=(n-i + 1);j++){
            process.stdout.write(`  `)
        }
        // for loop for star
        for(let k = 1;k<(2*i);k++){
            process.stdout.write(`* `)
        }
        process.stdout.write("\n");
    }
    process.exit();
    })